src/services/userService.js

```diff
@@ -0,0 +1,34 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const newMessageService = (newMessage, user, emiter) => {
+  const unicId = uuidv4();
+
+  if (!newMessage) {
+    return 'Message required';
+  }
+
+  if (!user) {
+    return 'user required';
```

Comment:
The error message 'user required' should be capitalized to maintain consistency with other error messages.